package.json

```diff
@@ -48,11 +48,13 @@
 		"watch:firefox": "yarn run set-manifest:firefox:force && yarn watch"
 	},
 	"dependencies": {
+		"ramda": "^0.29.1",
```

Comment:
Adding ramda doubles the `background.js` file size. Things like `difference` and `union` might be a bit annoying to write but a lot of the other functions like `map` and `filter` are available natively so I would consider if we actually need this.

With ramda / Without ramda
<img width="1295" alt="image" src="https://github.com/gitkraken/gk-browser-extension/assets/899916/baba7f22-e703-4cb9-a3cb-36bfb1817b64">

Reply:
Fair point, I think I only really need `difference` so I'll write a util func for that.